src/console/dbseed/domain/db-seed.service.ts

```diff
@@ -163,7 +165,11 @@ export class DbSeedService {
                 throw rolle;
             }
 
-            const persistedRolle: Rolle<true> = await this.rolleRepo.save(rolle);
+            if (file.overrideId) {
+                rolle.id = this.getValidUuidOrUndefined(file.overrideId);
+            }
+
```

Comment:
In line 111 you don't use an if, could we make this the same everywhere?

Reply:
yes